src/arguments.js

```diff
@@ -174,7 +175,23 @@ function getOptions(options) {
 
     // Set email note.
     commandOptions.note = options.note || process.env[ENV_VAR.EMAIL_NOTE];
+    if (commandOptions.note !== DEFAULT_EMAIL_NOTE && fs.existsSync(commandOptions.note)) {
+        commandOptions.note = fs.readFileSync(commandOptions.note, "utf8");
+    }
+    commandOptions.note = getHtml(commandOptions.note);
 
     spinner.succeed('Fetched argument values')
     return commandOptions;
+}
+
+// Convert text to html
+function getHtml(text) {
+    text = (text || "");
+    return text
+        .replace(/&/g, "&amp;")
+        .replace(/</g, "&lt;")
+        .replace(/>/g, "&gt;")
+        .replace(/\t/g, "    ")
+        .replace(/ /g, "&#8203;&nbsp;&#8203;")
+        .replace(/\r\n|\r|\n/g, "<br />");
```

Comment:
does this need `"` and `'`? (`&quot;` and `&apos;`)

Reply:
maybe not needed. tested with text file note 
<img width="399" alt="image" src="https://user-images.githubusercontent.com/56703525/214916095-38ae9ec8-b056-439d-ad28-b772d0875726.png">